operator/src/main/resources/scripts/commonMount.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Init container script for the common mount feature. 
+# See 'domain.spec.serverPod.commonMount' for details.
+
+scriptDir="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
+
+if [ "${debug}" == "true" ]; then set -x; fi;
+
+source ${scriptDir}/utils.sh
+[ $? -ne 0 ] && echo "[SEVERE] Missing file ${scriptDir}/utils.sh" && exit 1
+
+if [ -z ${COMMON_MOUNT_COMMAND} ]; then
+  trace ERROR "The common mount command $COMMON_MOUNT_COMMAND is empty. Exiting"
```

Comment:
'$COMMON_MOUNT_COMMAND' will resolve to '' in this case, and so isn't needed. 

Also, maybe make the error message a little more informative with "The 'serverPod.commonMount.container.mountCommand' is empty."

Finally, we don't know which image has the error, so maybe move the following trace above this trace.

Reply:
Two more things:
- Better yet, include the name of the container's image in the error message. 
- We probably don't want to exit 1 as  this would short circuit the checks/tracing to pod log in the main container.   
- Ditto for (all?) of the other exit 1 failures...

Reply:
Fixed.